### PR TITLE
Docs: Final edits, Style enhancements, Vale checks 

### PR DESCRIPTION
**Pull request by sheilamjones**
This PR fixes style issues (title capitalization, one sentence per line...), links, Vale issues, and some grammar as per the preferred Quarkus Contributor guidelines in the security-oidc-code-flow-authentication-tutorial.adoc (Protect a web application by using OpenID Connect (OIDC) authorization code flow guide)

Reference: QDOCS-570

### CODE REVIEW

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 1]
```diff
@@ -27,18 +27,18 @@ include::{includes}/prerequisites.adoc[]
...
 
 == Solution
 
-We recommend that you follow the instructions in the next sections and create the application step by step.
-However, you can go right to the completed example.
+Follow the instructions in the next sections and create the application step by step.
+Alternatively, you can go right to the completed example.
 
-Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+Clone the Git repository by using `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:41]:
```suggestion
Clone the Git repository by running a `git clone {quickstarts-clone-url}` command, or download an {quickstarts-archive-url}[archive].
```

I'm not sure it is totally correct from the language perspective, but `by using` reads a little bit unusual

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 2]
```diff
@@ -153,13 +154,13 @@ public class TokenResource {
 This endpoint has ID, access, and refresh tokens injected.
 It returns a `preferred_username` claim from the ID token, a `scope` claim from the access token, and also a refresh token availability status.
 
-Note that you do not have to inject the tokens - it is only required if the endpoint needs to use the ID token to interact with the currently authenticated user or use the access token to access a downstream service on behalf of this user.
+Note that you do not have to inject the tokens - it is only required if the endpoint needs to use the ID token to interact with the currently-authenticated user or use the access token to access a downstream service on behalf of this user.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:157]:
```suggestion
Note that you do not have to inject the tokens - it is only required if the endpoint needs to use the ID token to interact with the currently authenticated user or use the access token to access a downstream service on behalf of this user.
```

Introducing `-` reads unusual

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 3]
```diff
@@ -243,19 +243,20 @@ After getting a cup of coffee, you can run this binary directly:
...
-To authenticate to the application, type the following credentials when at the Keycloak login page:
+To authenticate to the application, enter the following credentials at the Keycloak login page:
 
 * Username: *alice*
 * Password: *alice*
 
 After clicking the `Login` button, you are redirected back to the application and a session cookie will be created.
 
-The session for this demo is short-lived and you will be asked to re-authenticate on every page refresh. Please follow the Keycloak https://www.keycloak.org/docs/latest/server_admin/#_timeouts[session timeout] documentation to learn how to increase the session timeouts. For example, you can access Keycloak Admin console directly from Dev UI by selecting a `Keycloak Admin` link if you use xref:security-oidc-code-flow-authentication.adoc#integration-testing-keycloak-devservices[Dev Services for Keycloak] in dev mode:
+The session for this demo is temporary and, on every page refresh, you will be asked to re-authenticate.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:257]:
```suggestion
The session for this demo is valid for a short period of time and, on every page refresh, you will be asked to re-authenticate.
```

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc (hunk 1 above)
Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:41]: Good point. Thank you. Updated per your suggestion.

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc (hunk 2 above)
Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:157]: I added the hyphen to join the words together as both together formed an adjective that modified the word "user". From a grammatical perspective, we would typically have a hyphen in this instance. However, I did some more research into these types of adjectives and both options are acceptable here. Therefore, I will remove in this instance again, per your suggestion.

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc (hunk 3 above)
Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:257]: Updated per your proposal.

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 4]
```diff
@@ -270,13 +271,13 @@ After you have completed this tutorial, explore xref:security-oidc-bearer-token-
...
+* xref:security-openid-connect-client-reference.adoc[OpenID Connect and OAuth2 Client and Filters reference guide]
 * xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak]
 * xref:security-jwt-build.adoc[Sign and encrypt JWT tokens with SmallRye JWT Build]
 * xref:security-authentication-mechanisms.adoc#oidc-jwt-oauth2-comparison[Choosing between OpenID Connect, SmallRye JWT, and OAuth2 authentication mechanisms]
 * xref:security-keycloak-admin-client.adoc[Quarkus Keycloak Admin Client]
 * https://www.keycloak.org/documentation.html[Keycloak Documentation]
 * xref:security-oidc-auth0-tutorial.adoc[Protect Quarkus web application by using Auth0 OpenID Connect provider]
 * https://openid.net/connect/[OpenID Connect]
-* https://tools.ietf.org/html/rfc7519[JSON Web Token]
+* https://tools.ietf.org/html/rfc7519[JSON Web token]
```
[the remark this answers is not in the document]
Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:283]: Thanks for the additional clairification. Updated per your suggestion.

**Comment by sberyozkin** [comment]
Thanks @sheilamjones, can you please push the latest updates ?

**Comment by MichalMaler** [comment]
@sberyozkin Hello there! I will review it soon and that we are good to go!
Will ping ya when ready :)

**Review by sberyozkin — APPROVED** [review]

**Comment by sberyozkin** [comment]
Hi @MichalMaler, sure, will wait, thanks

**Review by rolfedh — APPROVED** [review]
Looks good to me. Nice work.

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 5]
```diff
@@ -27,18 +27,19 @@ include::{includes}/prerequisites.adoc[]
 
 == Architecture
 
-In this example, we build a very simple web application with a single page:
+In this example, we build a simple web application with a single page:
 
 * `/index.html`
 
-This page is protected and can only be accessed by authenticated users.
+This page is protected and only authenticated users can access the page.
```
Comment (MichalMaler) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:34]:
```suggestion
This page is protected, and only authenticated users can access it.
```

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 6]
```diff
@@ -99,27 +100,28 @@ import io.quarkus.oidc.RefreshToken;
...
    RefreshToken refreshToken;
 
    /**
-    * Returns the tokens available to the application. This endpoint exists only for demonstration purposes, you should not
-    * expose these tokens in a real application.
+    * Returns the tokens available to the application.
+    * This endpoint exists only for demonstration purposes.
+    * Do not expose these tokens in a real application.
     *
     * @return a HTML page containing the tokens available to the application
```
Comment (MichalMaler) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:126]:
```suggestion
    * @return an HTML page containing the tokens available to the application.
```

**Comment by MichalMaler** [comment]
@sheilamjones 
Line 178: (missing comma)
The `quarkus.oidc.client-id` property references the `client_id` issued by the OIDC provider, and the `quarkus.oidc.credentials.secret` property sets the client secret.

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 7]
```diff
@@ -179,7 +181,7 @@ The `quarkus.oidc.application-type` property is set to `web-app` in order to tel
 
 Finally, the `quarkus.http.auth.permission.authenticated` permission is set to tell Quarkus about the paths you want to protect.
```
Comment (MichalMaler) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:182]:
```suggestion
Finally, the `quarkus.http.auth.permission.authenticated` permission is set to tell Quarkus about the paths you want to protect.
```

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 8]
```diff
@@ -179,7 +181,7 @@ The `quarkus.oidc.application-type` property is set to `web-app` in order to tel
...
 In this case, all paths are being protected by a policy that ensures that only `authenticated` users are allowed to access.
```
Comment (MichalMaler) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:183]:
```suggestion
In this case, all paths are being protected by a policy ensuring that only `authenticated` users can access them.
```

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 9]
```diff
@@ -195,18 +197,18 @@ where `keycloak.version` should be set to `23.0.0` or higher.
...
 
 == Run the application in dev and JVM modes
 
-To run the application in a dev mode, use:
+To run the application in dev mode, use:
 
 include::{includes}/devtools/dev.adoc[]
 
-When you're done playing with dev mode, you can run it as a standard Java application.
+When you are finished exploring the application in dev mode, you can run it as a standard Java application.
```
Comment (MichalMaler) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:211]:
```suggestion
After exploring the application in dev mode, you can run it as a standard Java application.
```

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 10]
```diff
@@ -243,24 +244,27 @@ After getting a cup of coffee, you can run this binary directly:
 
 To test the application, open your browser and access the following URL:
 
-
 * http://localhost:8080/tokens[http://localhost:8080/tokens]
 
 If everything is working as expected, you are redirected to the Keycloak server to authenticate.
```
Comment (MichalMaler) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:249]:
```suggestion
If everything works as expected, you are redirected to the Keycloak server to authenticate.
```

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 11]
```diff
@@ -243,24 +244,27 @@ After getting a cup of coffee, you can run this binary directly:
...
 
-To authenticate to the application, type the following credentials when at the Keycloak login page:
+To authenticate to the application, enter the following credentials at the Keycloak login page:
 
 * Username: *alice*
 * Password: *alice*
 
 After clicking the `Login` button, you are redirected back to the application and a session cookie will be created.
```
Comment (MichalMaler) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:256]:
```suggestion
After clicking the `Login` button, you are redirected back to the application, and a session cookie will be created.
```

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc [hunk 12]
```diff
@@ -243,24 +244,27 @@ After getting a cup of coffee, you can run this binary directly:
...
 
-The session for this demo is short-lived and you will be asked to re-authenticate on every page refresh. Please follow the Keycloak https://www.keycloak.org/docs/latest/server_admin/#_timeouts[session timeout] documentation to learn how to increase the session timeouts. For example, you can access Keycloak Admin console directly from Dev UI by selecting a `Keycloak Admin` link if you use xref:security-oidc-code-flow-authentication.adoc#integration-testing-keycloak-devservices[Dev Services for Keycloak] in dev mode:
+The session for this demo is valid for a short period of time and, on every page refresh, you will be asked to re-authenticate.
```
Comment (MichalMaler) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:258]:
```suggestion
The session for this demo is valid for a short period, and on every page refresh, you will be asked to re-authenticate.
```

**Comment by MichalMaler** [comment]
@sheilamjones @sberyozkin Hello!
Added few little suggestions. Looks great overall. Feel free to apply or merge. 
Cheers!

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc (hunk 12 above)
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:258]: All the above suggestions make sense IMHO but this one looks like an over optimization :-), `period of time` reads clearer to me

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc (hunk 7 above)
Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:182]: @MichalMaler - not sure what was changed here?

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc (hunk 9 above)
Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:211]: Reads much more smoothly, thanks for the suggestion!

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc (hunk 12 above)
Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:258]: Thank you both for your suggestions. I will keep this one as is then, that is, keep as "period of time"

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc (hunk 7 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:182]: Not sure how this get into creation of a diff. Feel free to close this one :)

Comment (rolfedh) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:182]: I believe this happens when the writer's editor is configured to trim whitespace from the end of the line (the standard practice), and GitHub is configured to hide whitespace changes.

Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication-tutorial.adoc:182]: Thanks for clarifying @rolfedh and thanks @MichalMaler also for your review and clarification.

**Comment by sheilamjones** [comment]
@sberyozkin, thanks again for your review. I believe this one is ready to merge.

**Review by MichalMaler — APPROVED** [review]

**Review by jmartisk — APPROVED** [review]

**Comment by jmartisk** [comment]
There seem to be some conflicts.. :/

**Comment by sheilamjones** [comment]
Hi @jmartisk, apologies, I'm not sure what happened there. Many thanks for looking at this PR. I have fixed all the conflicts and it is running through the final build checks now and should be ready then to merge. 
Many thanks,
Sheila

**Comment by jmartisk** [comment]
All looks good now, thanks

**Comment by sheilamjones** [comment]
Thanks so much @jmartisk. Much appreciated.

**Pull request merged**